src/Uno.Toolkit.UI/Controls/Chips/ChipSelectionMode.cs

```diff
@@ -12,10 +19,17 @@ public enum ChipSelectionMode
 		None,
 
 		/// <summary>
-		/// Only one item can be selected at a time.
+		/// Up to one item can be selected at a time.
 		/// </summary>
+		/// <remarks>The current item can be deselected.</remarks>
 		Single,
 
+		/// <summary>
+		/// One item is selected any time.
+		/// </summary>
+		/// <remarks>The current item cannot be deselected.</remarks>
+		SingleEnforced,
```

Comment:
this is really bad name, we have any better idea?

Reply:
I'm curious what we'd need the SingleEnforced for?

Reply:
#354

Reply:
Maybe `SingleOnly`? Or maybe `Single` could be `SingleOrNone` and then `Single` would now be the one where you can't deselect?

In a ListView, can you de-select the selected item on click by default?

Reply:
uwp: `Single` mode is that radiogroup-like behavior; you cant deselect
good idea, i will use `Single` and `SingleOrNone`, and mark this as breaking change
this should be a welcome change